salt/modules/virt.py

```diff
@@ -5455,7 +5456,8 @@ def _volume_extract_infos(vol):
                 'used_by': used_by,
             }
 
-        pools = [obj for obj in conn.listAllStoragePools() if pool is None or obj.name() == pool]
+        pools = [obj for obj in conn.listAllStoragePools()
+                    if (pool is None or obj.name() == pool)  and obj.info()[0] == 2]
```

Comment:
The same is here.

Reply:
Also lint found there are 2 spaces before `and` here.

Reply:
fixed